src/modules/settings/styles/_rsuite.less

```diff
@@ -20,6 +20,20 @@
   width: 800px;
   height: 500px;
 
+  color: var(--bttv-rs-text-primary);
+  font-size: 1.3rem;
+  line-height: 1.5;
+
+  p {
```

Comment:
we should not set styles for html elements. in general this will cause pain. the goal of css modules is to prevent styles from bleeding between components.